File: src/main/kotlin/de/hbrs/team7/se1_starter_repo/Car.java
package de.hbrs.team7.se1_starter_repo;

import java.util.Arrays;

public class Car implements CarIF {
    String[] params;
    public Car( String[] params ){
        this.params = params;
    }

    @Override
    public int nr() {
        return 0;
    }

    @Override
    public long begin() {
        return 0;
    }

    @Override
    public long end() {
        return 0;
    }

    @Override
    public int duration() {
        return 0;
    }

    @Override
    public int price() {
        return 0;
    }

    @Override
    public String toString(){
        return Arrays.toString( params );
    }
}

File: src/main/kotlin/de/hbrs/team7/se1_starter_repo/Car.kt
package de.hbrs.team7.se1_starter_repo

class Car(var params: Array<String>) : CarIF {

    override val nr: Int
        get() = 0

    override val begin: Long
        get() = 0

    override val end: Long
        get() = 0

    override val duration: Int
        get() = 0


    override var price: Int = 0

    override var type: String = ""


    override fun toString(): String {
        return params.contentToString()
    }

    override fun paramsToArray(): Array<String>? {
        return params
    }

}
